xep-0065.xml

```diff
@@ -30,6 +30,16 @@
   &stpeter;
   &infiniti;
   <revision>
+    <version>next</version>
+    <date>201?-??-??</date>
+    <initials>fs</initials>
+    <remark>
+      <ul>
+        <li>Fixed description of activate element in 9.4.</li>
+      </ul>
+    </remark>
+  </revision>
+  <revision>
```

Comment:
Actually, per discussion in the Editor Team, we decided to create "patch-point releases" for fixes like this: http://logs.xmpp.org/editor/2015-09-15/#15:52:08